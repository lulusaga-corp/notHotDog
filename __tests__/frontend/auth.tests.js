import React from 'react';
import requireAuth from '../../src/scenes/auth/requireAuth';
import requireNotAuth from '../../src/scenes/auth/requireNotAuth';
import Signup from '../../src/scenes/auth/Signup';
import Signin from '../../src/scenes/auth/Signin';

import renderer from 'react-test-renderer';

/* Auth Containers */
describe('requireAuth', () => {
  it('renders without crashing', () => {
    const renderedRequireAuth = renderer.create(<requireAuth />).toJSON();
    expect(renderedRequireAuth).toMatchSnapshot();
  })
})

describe('requireNotAuth', () => {
  it('renders without crashing', () => {
    const renderedRequireNotAuth = renderer.create(<requireNotAuth />).toJSON();
    expect(renderedRequireNotAuth).toMatchSnapshot();
  })

  it('renders signup component', () => {
    const renderedSignup = requireNotAuth(Signup);
    expect(renderedSignup).toMatchSnapshot();
  })

  it('renders signin component', () => {
    const renderedSignin = requireNotAuth(Signin);
    expect(renderedSignin).toMatchSnapshot();
  })
})