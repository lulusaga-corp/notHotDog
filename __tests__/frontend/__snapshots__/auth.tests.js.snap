// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`requireAuth renders without crashing 1`] = `<requireAuth />`;

exports[`requireNotAuth renders signin component 1`] = `[Function]`;

exports[`requireNotAuth renders signup component 1`] = `[Function]`;

exports[`requireNotAuth renders without crashing 1`] = `<requireNotAuth />`;
